packages/plugin-core/src/_extension.ts

```diff
@@ -353,7 +354,14 @@ export async function _activate(
         const ONE_MINUTE_IN_MS = 60_000;
         setTimeout(() => {
           const showcase = new FeatureShowcaseToaster();
-          showcase.showToast();
+          // Temporarily show the new toast instead of the rest.
+          // for subsequent sessions this will not be shown as it already has been shown.
+          const hasShown = showcase.showSpecificToast(
```

Comment:
can we add to the show criteria that we only show this to users who did not just install vscode?

so as part of this change, add a `firstInstallVersion` as part of the install? if `firstInstallVersion` is not undefined or less than `0.112.0`, then show this message

Reply:
> if firstInstallVersion is not undefined or less than 0.112.0, then show this message

Do you mean if `firstInstallVersion` is _undefined_ or less than 0.112.0? unless we go and backfill them based on the `firstInstall` time one by one, all existing users will have them undefined.

Reply:
the former